src/endpoint/s3/s3_rest.js

```diff
@@ -14,6 +14,7 @@ const http_utils = require('../../util/http_utils');
 const signature_utils = require('../../util/signature_utils');
 const config = require('../../../config');
 const s3_utils = require('./s3_utils');
+const { _create_detailed_message_for_access_in_s3 } = require('../iam/iam_utils'); // authorize_request for IAM policy
```

Comment:
Is this comment needed?

Reply:
I thought it can help others, if you think not, I will remove it (just let me know).